console/billing/how-to/manage-payment-information.mdx

```diff
@@ -0,0 +1,66 @@
+---
+meta:
+  title: How to manage your payment information
```

Comment:
I think you can change the name of this how-to to: "How to add a payment method"